server/src/main/java/org/candlepin/model/Release.java

```diff
@@ -70,7 +72,9 @@ public boolean equals(Object anObject) {
 
     @Override
     public int hashCode() {
-        return releaseVer.hashCode();
+        HashCodeBuilder hcBuilder = new HashCodeBuilder();
+        hcBuilder.append(releaseVer);
+        return hcBuilder.toHashCode();
```

Comment:
The hashcodebuilder takes care of the case releaseVer is null, otherwise the existing code throws a NPE. Should I not keep the builder and seed it with primes?

Reply:
Until we have additional values to serialize (which we never will), it doesn't give us any value to make the hash code more complex. If the null check is a concern, this should become:

```
return this.releaseVer != null? this.releaseVer.hashCode() : 0
```